hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
```

Comment:
Shouldn't be G400.

Reply:
This will not be the G400. Gus would not let me get pics to put in its place. I had to use what was there for the skeleton. Either Greg needs to make them for me or I will need to take temporary images that I will use as a place holder.

Reply:
You can change the url to what it needs to be even though it won't show up, it'll be a build warning that we can use as a reminder to get one.

Reply:
ok. I will replace.